common/video.js

```diff
@@ -27,9 +27,6 @@ class Video {
 			delete this.description;
 			delete this.url;
 		}
-		else if (["direct"].includes(this.service)) {
```

Comment:
I changed a few things here and there to make things work and I probably broke *something*. This is one of those places.

I don't understand why you deleted these properties instead of just ignoring them. I built the direct video adapter so that it stores the URL in the ID property through the `getVideoId` method. That didn't work, however, because the Video class would remove the `id` property and the URL would get lost. That's why I removed this.